Cargo.toml

```diff
@@ -18,7 +18,11 @@ features = ["std"]
 
 [dependencies]
 
+[build-dependencies]
+rustc_version = "0.2"
+
 [features]
 default = ["std"]
 std = []
 i128 = []
+int_to_from_bytes = []
```

Comment:
I don't think we need a crate feature for this. The i128 feature is sort of vestigial, as I added that before I figured out how to automatically probe for it.

Reply:
ok, I agree